### PR TITLE
Pruned repo

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: database/api/routes.md
Comment (aniehuser) [inline database/api/routes.md]: we should keep this file, usually api's have some sort of documentation

File: scripts/serverSetup.md
Comment (aniehuser) [inline scripts/serverSetup.md:1]: is this information located somewhere else?

Comment (clundin55) [reply scripts/serverSetup.md:1]: Yea most of this is redundant with the README

**Review by aniehuser — CHANGES_REQUESTED** [review]
we shouldnt merge this if we are deleting the API documenation

File: database/api/routes.md
Comment (clundin55) [reply database/api/routes.md]: I added this back

**Pull request merged**
